TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts

```diff
@@ -19,7 +19,7 @@ var props = Properties().apply {
 var dafnyVersion = props.getProperty("dafnyVersion")
 
 group = "software.amazon.cryptography"
-version = "1.5.1-SNAPSHOT"
+version = "1.0-SNAPSHOT"
```

Comment:
I am confused. Why is this version 1.0?

Reply:
the test vectors project is unpublished.
It's version used to be 1.0-SNAPSHOT but an update to the releaserc.cjs file bumped this as well, it should not have.

The DBESDK and ESDK depend on this project as 1.0-SNAPSHOT  not 1.5.1-SNAPSHOT as a result if we try to build the latest test vectors in the dbesdk it fails because it is looking for 1.0-SNAPSHOT